csharp/extractor/Semmle.Extraction.CSharp/Entities/Expression.cs

```diff
@@ -42,7 +45,7 @@ internal Expression(IExpressionInfo info)
             if (info.ExprValue is string value)
                 cx.Emit(Tuples.expr_value(this, value));
 
-            Type.ExtractGenerics();
+            Type.Type.ExtractGenerics();
```

Comment:
Should there be a call to (an equivalent of) `ExtractNullability` here?

Reply:
Unfortunately, that information does not seem to be available now. I expect it will come soon in a future version of Roslyn.

Reply:
OK.